apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types-listing.module.ts

```diff
@@ -1,23 +1,40 @@
 import { NgModule } from '@angular/core';
```

Comment:
create a `dot-content-type-clone-dialog.module.ts` and add that dependency here, just like we do with `DotAddToBundleModule`

Reply:
Module created.